ldk/javascript/src/types/oliveHelps/user.d.ts

```diff
@@ -16,7 +16,6 @@ declare namespace User {
   }
 
   interface JWTWithUserDetails {
-    email?: string;
     fullName?: string;
```

Comment:
Did u accidentally delete this email prop? 🤔

Reply:
That's from Brett's commit here https://github.com/open-olive/loop-development-kit/pull/509/commits/3a5feb2649ae1aa2ea89ec060059a56d6bd33be0